workspace-extension/src/extension.ts

```diff
@@ -147,8 +188,65 @@ export function activate(context: vscode.ExtensionContext) {
         } finally {
           cleanup.forEach((f) => f());
         }
-      }
-    )
+      },
+    ),
+  );
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand(
+      "remote-udp-tunnel.forwardPortRange",
+      async (targetParam?: unknown) => {
+        let targets = await resolveTargetRangeParam(targetParam);
+        for (let target of targets) {
+          let proxy = await getTcpReverseProxyForUdp(target);
```

Comment:
What do you think of extracting a shared function between this loop body and the 1 port case?

Reply:
Could be done, but I didn't really know the code well enough to be sure I didn't break anything.

Reply:
I'll take a crack at it if you don't get to it. I wanna say it shouldn't be too dangerous.